orchestrator/config.js

```diff
@@ -113,3 +120,48 @@ module.exports = {
         level: process.env.LOG_LEVEL || 'info' // it could be error, warn, info or debug
     }
 };
+
```

Comment:
Is there code in development?

Reply:
Hi @mprevide, there is no code in development.

Reply:
Shouldn't this code be there too?